src/lib/manifest.ts

```diff
@@ -0,0 +1,58 @@
+import path from "path"
+import fs from "fs"
+import chalk from "chalk"
+
+const { CDN_URL, NODE_ENV } = process.env
+
+class ManifestManager {
+  private loadError = false
+  private manifest = {}
+
+  constructor(private manifestFileName = "manifest.json") {
```

Comment:
How does that `private manifestFileName` arg work? Haven't seen this before

Reply:
seconded! This is super interesting.

Reply:
TypeScript supports constructor assignment, public and private, so that you don't need to do the following:

```ts
class TestClass {
  private manifestFileName: string;

  constructor(manifestFileName: string) {
    this.manifestFileName = manifestFileName;
  }
}
```